Plot output and gradients of activations and quantizers in docs

lgeiger opened the pull request:
This adds simple matplotlib plots to our documentation, which will also add an additional sanity check for the math:
<img width="1552" alt="Bildschirmfoto 2019-04-23 um 21 06 53" src="https://user-images.githubusercontent.com/13285808/56612325-fdc91d00-660b-11e9-995f-f9effbba49bf.png">

koenhelwegen commented:
Awesome!! This is really cool. Small request: would it be possible to add titles? E.g. "Forward pass (activation)" and "Backward pass (gradient)" or something like that? The dy/dx is a bit too subtle I think.

lgeiger commented:
Added titles and modified the font to match the docs theme. It's not perfect should look reasonable:
![Bildschirmfoto 2019-04-24 um 10 17 16](https://user-images.githubusercontent.com/13285808/56647904-2b4caf80-667a-11e9-831c-b496db25b756.png)
